functions/weather.location.fish

```diff
@@ -6,36 +6,34 @@ function weather.location -d "Get the current geographic location"
   end
 
   # Fetch location data based on our IP
-  if not set geoip_data (weather.fetch "http://geoip.nekudo.com/api/$ip")
+  if not set geoip_data (curl -s "https://ipapi.com/ip_api.php?ip=$ip")
     echo "Unable to query GeoIP data; please try again later."
     return 1
   end
-
+  
   # Echo coordiantes.
-  echo $geoip_data | jq '.location.latitude'
-  echo $geoip_data | jq '.location.longitude'
+  echo $geoip_data | jq '.latitude'
+  echo $geoip_data | jq '.longitude'
   echo $geoip_data | jq -r '.city?'
-  echo $geoip_data | jq -r '.country.name'
+  echo $geoip_data | jq -r '.country_name'
 end
 
 function __weather_get_ip -d "Get the current device's public IP address"
   # Attempt to get our external IP using the OpenDNS resolver.
-  if set -q __weather_system_dns
-    if not set ip (dig +short myip.opendns.com)
-      return 1
-    end
-  else
-    if not set ip (dig +short myip.opendns.com @resolver1.opendns.com)
-      return 1
-    end
-  end
+  # if set -q __weather_system_dns
+  #  if not set ip (dig +short myip.opendns.com)
+  #    return 1
+  #  end
+  # else
+  #  if not set ip (dig +short myip.opendns.com @resolver1.opendns.com)
+  #    return 1
+  #  end
+  # end
 
   # Attempt to get our external IP using a web service.
-  if test -z $ip
-    if not set ip (weather.fetch "http://ipecho.net/plain")
-      return 1
-    end
+  if not set ip (weather.fetch "http://ipecho.net/plain")
+    return 1
   end
-
+  
```

Comment:
Please remove this from the diff, it's not part of the fix.

Reply:
This structures the code.

Reply:
Save some cases (e.g. Markdown) traling spaces are unnecessary and just polute the code/markup.